corehq/apps/userreports/tasks.py

```diff
@@ -85,6 +85,16 @@ def rebuild_indicators(indicator_config_id, initiated_by=None, limit=-1, source=
         send = toggles.SEND_UCR_REBUILD_INFO.enabled(initiated_by)
     with notify_someone(initiated_by, success_message=success, error_message=failure, send=send):
         adapter = get_indicator_adapter(config)
+
+        if engine_id:
+            if getattr(adapter, 'all_adapters', None):
```

Comment:
does this mean that  you can't trigger rebuilds through the UI since you can't specify engine_id (that's fine for me, but wnted to verify)

Reply:
No, if it's triggered through the UI it won't include `engine_id` so it won't do any check and just call `adapter.rebuild()` on the `MultiDBSqlAdapter` which will rebuild all tables.

Reply:
Will the `engine_id` be ever other than the one corresponding to `adapter.main_adapter`?

Reply:
Yes it could which is exactly the scenario we have now - dbA has all tables created, dbB is empty and we set adapter to mirror to dbB so need to build the table on dbB but not on dbA.

Reply:
👍